packages/devtools_app/lib/src/screens/debugger/variables.dart

```diff
@@ -58,13 +57,13 @@ class ExpandableVariable extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    if (variable == null) return const SizedBox();
+    final _variable = variable;
```

Comment:
Starting local variable with '_' looks confusing for me as normally '_' means visibility that does not apply to local variables. Can you name it variable_  or variableLocal instead?

Reply:
renamed.